ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -1559,9 +1559,17 @@
                 "\n## Data Node Parameters",
                 "\n",
                 "\nData Node Parameters are used to keep track of other parameters pertaining to each Data Node, and their movements in time (year and month).",
+                "\n",
                 "\n<br> <code>PremiumAllocation</code> : defines the weight of Premium to be included in the Experience Adjustement AoC Type of the Technical Margin and is valid only for Group of Insurance Contract with LiabilityType : Liability for Remaining Coverage.",
+                "\n",
                 "\n<br> <code>CashFlowPeriodicity</code> : defines the periodicity of the provided cash flows, it is not a mandatory input. Supported values can be found [here](../Constants/Enums#cashflowperiodicity). When the column *CashFlowPeriodicity* is missing from the input file it is assumed a monthly periodicity (default value). When the *CashFlowPeriodicity* column is present, a valid value must be entered. ",
+                "\n",
                 "\n<br> <code>InterpolationMethod</code> : defines the interpolation method to be applied to interpolate the cash flow to the monthly granularity, it is not a mandatory input. Supported values can be found [here](../Constants/Enums#interpolation-method). When the *InterpolationMethod* is not present in the input file or the *CashFlowPeriodicity* has the default value, then the default value for the *InterpolationMethod* is applied. The default value is Not Applicable. ",
+                "\n",
+                "\n<br> <code>Amortization Pattern</code> : defines the pattern of factors that will be used to defer and amortize any AmountType over the course of contract term. It is not a mandatory input.  Note this pattern is the fallback if no *Amortizaltion Pattern* is provided in the Cashflows, specified by AmountType and EstimateType.",
```

Comment:
rename Amortizaltion Pattern

Reply:
Add that this pattern should always be provided with Value0 corresponding to the annual cohort